types/stripe/index.d.ts

```diff
@@ -3835,8 +3838,9 @@ declare namespace Stripe {
              * will attempt to pay this invoice using the default source attached to the customer.
              * When sending an invoice, Stripe will email this invoice to the customer with payment
              * instructions. Defaults to charge_automatically.
+             *
              */
-            billing?: 'charge_automatically' | 'send_invoice';
```

Comment:
Is this a recent API change or should it always have been `collection_method`?  If it's an API change the old REST API is still usable with the latest library version so the old param should still be available but `@deprecated`.

Reply:
fixed